### PR TITLE
Add blog post for August 25, 2022 Team of Teams panel

### PR DESCRIPTION
**Pull request by mrmundt**
# Description

EB Member: @bernhold 

Resolves #NA

This is a blog post for the August 25th panel "What can be learned from applying Team of Teams principles?" as part of the Strategies for Working Remotely webinar series.

_NOTE_: Currently working on getting a Hero Image made for the post.

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* ~~[ ] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.~~
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [ ] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [ ] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [ ] ***[EB Mem]*** Add one or more Reviewers.
* [ ] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [ ] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [ ] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [ ] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [ ] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [ ] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [ ] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [ ] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [ ] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [ ] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [ ] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

[preview]: https://preview.bssw.io
[bssw.io]: https://bssw.io
[Content Development]: https://github.com/betterscientificsoftware/bssw.io/projects/3
[BSSw Internal]: https://github.com/betterscientificsoftware/bssw.io/projects/2
[meta-data]: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_common.html#metadata-section
[wikize_refs.py]: https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy

### CODE REVIEW

**Comment by bernhold** [comment]
@mrmundt I see a lot of bios that aren't on the contributor list.  What's your thinking there?  I feel like there should be a 1:1 correspondence.  Thanks

**Comment by mrmundt** [comment]
@bernhold - I agree, it's a long list of biographies; however, I would say in this situation, it makes sense since they were the highlight of the post and their biographies are not inherently explained within the post itself.

File: Articles/Blog/2022-08-Team-of-Teams-Panel.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+member of the IDEAS project. In all of these roles, he is active in the research of
+software sustainability and the application of software engineering
+methodologies for high-performance computational science. James received a
+M.S. in Computer Science from St. Cloud State University, and is currently a
+Software Engineering Ph.D. Candidate at North Dakota State University.
+
+Ulrike Yang is the Mathematical Algorithms and Computing group lead in the
+Center for Applied Scientific Computing at LLNL. She leads the xSDK
+project in the ECP and the Linear and Nonlinear Solvers topic area in
+the SciDAC-5 FASTMath Institute. She is also a software developer for the high
```
Comment (mrmundt) [inline Articles/Blog/2022-08-Team-of-Teams-Panel.md:289]:
```suggestion
the SciDAC-5 FASTMath Institute. She is also a software developer for the
```

File: Articles/Blog/2022-08-Team-of-Teams-Panel.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+performance solvers library hypre, which received an R&D 100 award in 2007. Her
```
Comment (mrmundt) [inline Articles/Blog/2022-08-Team-of-Teams-Panel.md:290]:
```suggestion
high-performance solvers library hypre, which received an R&D 100 award in 2007. Her
```

File: Articles/Blog/2022-08-Team-of-Teams-Panel.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+research interests are numerical algorithms, particularly iterative linear
+system solvers and algebraic multigrid methods, high performance computing,
```
Comment (mrmundt) [inline Articles/Blog/2022-08-Team-of-Teams-Panel.md:292]:
```suggestion
system solvers and algebraic multigrid methods, high-performance computing,
```

File: Articles/Blog/2022-08-Team-of-Teams-Panel.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+parallel algorithms, performance evaluation and scientific software design.
+She has a Ph.D in Computer Science from the University of Illinois at
```
Comment (mrmundt) [inline Articles/Blog/2022-08-Team-of-Teams-Panel.md:294]:
```suggestion
She has a Ph.D. in Computer Science from the University of Illinois at
```

**Comment by mrmundt** [comment]
Besides the few tiny changes, I think it looks great! Thanks, @bernhold !

**Pull request merged**